barrels/LogMonkey/LogTests.mc

```diff
@@ -0,0 +1,246 @@
+using Toybox.Test as Test;
+using Toybox.Lang as Lang;
+using LogMonkey as Log;
+
+module LogMonkey {
+
+    // Wrap the unit tests in a module annotated with :test so it will get compiled
+    // out for non unit test builds.
+    (:test)
+    module Test {
+
+        //! This class mocks the Toybox.System.println() interface but instead
+        //! of printing a value saves it to a string value which can be retrieved
+        //! and examed to verify log results are correct.
+        class MockSys {
+
+            private var mStreamContents;
+
+            //! A mock of Toybox.System.println() function
+            //! @param [Toybox::Lang::Object] message The message to print/log
+            public function println(message) {
+                if (mStreamContents == null) {
```

Comment:
If we initialize mStreamContents to an empty string, we can avoid this conditional.

Reply:
Done